src/less/less/ds6/mixins.less

```diff
@@ -102,3 +102,52 @@
     height: 17px;
     width: 17px;
 }
+
+// Element width in percentage - (No of gutters * gutter width / number of elements)px
+.create-grid(@cols, @gutter: 0, @size: 0, @min: 0, @max: 0) {
```

Comment:
So, this isn't really a "grid", but more of calculation of the width(s) of the container, based on the number of "columns", the gutter widths, and such?

If so, `create-grid` doesn't feel like the right way to term it. Maybe `create-grid-container-widths`? I know it's verbose, but the former seems a little generic.

Thoughts, @DylanPiercey?

@ianmcburnie Are we planning any kind of LESS mixins for the grid system? I think it would be wonderful for developers to have access to something that allowed them to plug into the [grid defined in the DSL (see Grid)](https://dsl.ebay.com/ds6/layout-spacing).

Reply:
It would be wonderful. But not yet on our roadmap. We can retrofit as necessary.

Reply:
Yup, I think we can tackle it soon.

Reply:
Sure, was just a question. ;-)

Reply:
@seangates I agree regarding the naming. Not sure whats best perhaps one of the following?

```
.fit-in-grid
.apply-grid-sizing
```

I'm not sure the `create` prefix even makes sense in this mixing since it's not really creating a grid as you say but applying the widths.